setup.py

```diff
@@ -1,31 +0,0 @@
-import io
-import os
-
-from setuptools import find_packages, setup
-
-
-def read(fname):
-    file_path = os.path.join(os.path.dirname(__file__), fname)
-    return io.open(file_path, encoding="utf-8").read()
-
-
-exec(open("pyflow/version.py").read())
-
-
-setup(
```

Comment:
I would keep a trivial `setup.py` for compatibility with older versions of `pip` when using editable installs. It should be enough to have
```python
from setuptools import setup
setup()
```

Reply:
Sounds good to me. Will update this PR with your feedback @oiffrig , thanks!!

Reply:
Done.